### PR TITLE
close stream after running tests

### PR DESCRIPTION
**Pull request by opensrcken**
This will also probably address https://github.com/sindresorhus/gulp-ava/issues/8.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -32,5 +32,7 @@ module.exports = function () {
 
 			gutil.log('gulp-ava:\n' + stderr);
 		}.bind(this));
+		
+		cb();
```
Comment (sindresorhus) [inline index.js]: This should be on line 32, otherwise it has no effect.

**Comment by sindresorhus** [comment]
Can you give the PR a proper title?

**Comment by opensrcken** [comment]
Done.

File: index.js [hunk 2]
```diff
@@ -29,7 +29,8 @@ module.exports = function () {
 				this.emit('error', new gutil.PluginError('gulp-ava', stderr));
 				return;
 			}
-
+		
+		        cb();
```
Comment (sindresorhus) [inline index.js]: Wrong indentation.

Comment (opensrcken) [inline index.js]: Ah, I guess you mean tab indents.

**Pull request closed without merging**